### ISSUE TITLE
hamburger button non-functional with bootstrap3 and USE_CDN=False

### ISSUE THREAD

**Issue by rbrewer123** [getnikola/nikola#2065]
I'm running Nikola 7.6.4 from the Debian packages.  If I create a fresh site with `nikola init -d site_testing`, accept all the defaults and build the site, I get a non-functional hamburger button (tested on Chrome and Firefox).  If I set `USE_CDN=True` and then rebuild the site, the hamburger button functions correctly.

**Comment by Kwpolska** [comment]
Cannot reproduce.

If you are testing the page using the `file://` protocol, it might break.  Use `nikola serve` instead. (newer versions of Nikola do not even work over `file://`)

Can you reproduce this with `nikola serve`?

**Comment by rbrewer123** [comment]
Thanks for trying that.  I was using `nikola serve` for my testing, and also had it deployed to Amazon S3 at one point with the same results.  I'm running nikola in a docker container, it's possible something is weird about that environment.  It's a minimal debian sid with only the nikola package installed, see here:  https://github.com/rbrewer123/docker_nikola/blob/master/Dockerfile

I'll do some more tests to try and understand what is going on.  Just to help narrow my search, were you trying to reproduce using the debian packages?

**Comment by Kwpolska** [comment]
No, I weren’t trying the Debian packages nor your Dockerfile (I’m not a fan of either, TBH), but I don’t think that’s a problem. You should, however, check what resources are requested and what are missing (with your web browser’s inspector feature).  Could you send me (chris@chriswarrick.com) a .zip archive of your output?

**Comment by rbrewer123** [comment]
I looked at the dev console in Chromium and it says "Uncaught Error: Bootstrap's JavaScript requires jQuery".  The nikola debian package depends on libjs-jquery and that is installed inside the docker container.  I'm not too familiar with debugging javascript issues but I'll keep poking around.

**Comment by rbrewer123** [comment]
It appears the files `/usr/lib/python2.7/nikola/data/themes/bootstrap3/assets/js/jquery.colorbox-min.js`, `jquery.min.js`, and `jquery.min.map` are missing from the debian 7.6.4 package.  When I did `pip install -U nikola[extras]` I received version 7.7.0 along with those files.  So I think something is wrong with the debian package.

**Comment by rbrewer123** [comment]
The debian package has this bug https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=791843 filed against it which seems to be the problem I'm experiencing.  Not sure if we should keep this issue open against Nikola itself.  Although perhaps a note on the Nikola getting started page could recommend against using the debian package until it is fixed.

**Comment by Kwpolska** [comment]
This bug actually is caused by two things:

 * Debian messing with our sources too much to deduplicate files (up until recently they refused to include the Bootstrap 3 theme due to some crazy packaging ideals of theirs)
 * We deleted the `bootstrap` theme in v7.6.1.  Guess who didn’t notice and thus failed to move their deduplication symlinks to `bootstrap3`?

> Although perhaps a note on the Nikola getting started page could recommend against using the debian package until it is fixed.

Well, we’ve had countless issues with them.  I added a note that discourages that package.  (though it’s v7.6.4, which is unusually up-to-date for Debian standards)

Closing as someone else’s bug/won’t fix.

Thanks for bringing this to our attention.
